### PR TITLE
Sp 1132: Replace FileSet adapters with FileStream adapters in all e2e tests

### LINKED ISSUE #1132: Update e2e tests to use FileStream adapter instead of FileSet
## Description
The current e2e tests use FileSet adapters to validate the results of the tests. However, since we want to remove the sets, the tests shoud rely on the new FileStream adapter (see #1121).

## Acceptance Criteria
The e2e tests should produce the same results when using the new FileStream adapter as they do when using the FileSet adapters

### StreamPipes Committer

I acknowledge that I am a maintainer/committer of the Apache StreamPipes project.

### PR DESCRIPTION
**Pull request by tenthe**
### Purpose

This PR fixes #1132 by removing all File Set adapter references and replacing them with a file stream adapter

### Remarks

PR introduces (a) breaking change(s): no

PR introduces (a) deprecation(s): no

### CODE REVIEW

File: ui/cypress/fixtures/connect/aggregationRules/expected.csv [hunk 1]
```diff
@@ -1,2 +0,0 @@
-timestamp;value
```
Comment (bossenti) [inline ui/cypress/fixtures/connect/aggregationRules/expected.csv:1]: Is this file intended to be empty?

**Review by bossenti — APPROVED** [review]

**Review by dominikriemer — APPROVED** [review]
